test/test.js

```diff
@@ -0,0 +1,49 @@
+﻿const mocha = require('mocha');
+const chai = require('chai');
+const program = require('../words');
+const assert = chai.assert;
+chai.config.truncateThreshold = 0;
+describe('Test words function', () => {
+    it('should handle specified input', () => {
```

Comment:
The wording of the describe block could be more descriptive. Something like: "It should return the words of an input sentence and their frequencies as key and value respectively of a JSON object."

It doesn't have to be as long and complicated as what I wrote, but at least, it should be descriptive enough for someone who probably doesn't know much about your function/code in the first place. The same applies to the other` describe` and `it` blocks.

Reply:
Thanks a lot. Will make necessary changes asap